torch/csrc/jit/register_special_ops.cpp

```diff
@@ -35,20 +35,35 @@ RegisterOperators reg({
           pack(stack, std::move(result));
           return 0;
         }),
+    // reference _list_with_default in utils.py
     Operator(
-        "aten::list_with_default(int[] list, int[] defaults) -> int[]",
+      "aten::list_with_default(int list, int[] defaults) -> int",
+      [](Stack& stack) {
+        IValue single_val;
+        IValue defaults;
+        pop(stack, single_val, defaults);
+        push(stack, single_val);
+        return 0;
+      }),
+    Operator(
+        "aten::list_with_default(int?[] list, int[] defaults) -> int[]",
         [](Stack& stack) {
           autograd::profiler::RecordFunction record("sizes");
```

Comment:
This can be removed

Reply:
even if it's not used in current context, i think it's good to match the semantics of the original function